### ISSUE TITLE
Slide number background fails accessibility guidelines

### ISSUE THREAD

**Issue by nigelmegitt** [hakimel/reveal.js#2737]
https://github.com/hakimel/reveal.js/blob/15815efe05ca69c35ce66cfdbf93316e1db66ecb/css/reveal.scss#L592 sets the background colour of the slide number to black with opacity to 0.4, and the colour to white. When the theme slide background is white, or close to white, perhaps `#F4F1F4`, this fails accessibility guidelines, due to the contrast being too low. Firefox's accessibility dev tool complains about it, to name just one.

Suggest increasing the opacity to 0.6.
